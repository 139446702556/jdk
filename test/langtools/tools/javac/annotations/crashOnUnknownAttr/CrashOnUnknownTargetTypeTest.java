/*
 * @test /nodynamiccopyright/
 * @summary compiler is crashing with AssertionError for annotations with unknown target type
 * @bug 8296010
 * @build A
 * @compile/fail/ref=CrashOnUnknownTargetTypeTest.out -XDrawDiagnostics CrashOnUnknownTargetTypeTest.java
 */

public class CrashOnUnknownTargetTypeTest {
    @A Object o;
}
