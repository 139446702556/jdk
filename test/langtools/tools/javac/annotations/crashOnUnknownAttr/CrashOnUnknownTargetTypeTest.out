- compiler.warn.unknown.enum.constant: A.class, java.lang.annotation.ElementType, NO_SUCH
- compiler.warn.unknown.enum.constant: Annotation.class, java.lang.annotation.ElementType, NO_SUCH
CrashOnUnknownTargetTypeTest.java:10:6: compiler.err.annotation.unrecognized.attribute.name: A, NO_SUCH
1 error
2 warnings
